databases/aws-rds/manifest.yml

```diff
@@ -7,19 +7,5 @@ applications:
       GOPACKAGENAME: aws-rds
       CGO_CFLAGS: -Isrc/github.com/cloud-gov/databases/aws-rds/include
       LD_LIBRARY_PATH: "/home/vcap/app/include/oracle:$LD_LIBRARY_PATH"
-      SERVICE_NAME: ((pg-service))
-      DB_TYPE: postgresql
-    services:
-      - rds-test-pg
```

Comment:
How do we know this is valid to remove?

Reply:
It never was used. The services are created by each of the tests with a specific name to indicate which test created the service. I've been testing without it this whole time. I will test again once this is merged.